pages/templates/pages/microsite_page.html

```diff
@@ -1,12 +1,210 @@
+        {% image page.custom_footer_background_image original as secondary_img %}
+        background-image: url({{ secondary_img.url }});
+      {% endif %}
+    }
+
+    #footer a, .microsite .footer a {
+      {% if page.custom_footer_text_color %}
+        color: #{{ page.custom_footer_text_color }};
+      {% endif %}
+    }
+
+    .microsite-content .can_button, .microsite-content #can_embed_form input[type="submit"], .microsite-content #can_embed_form .button, .microsite-content #donate_auto_modal input[type="submit"], .microsite-content #donate_auto_modal .button {
+      {% if page.button_background_color %}
+        background-color: #{{page.button_background_color}} !important;
+      {% endif %}
+
+      {% if page.button_text_color %}
+        color: #{{page.button_text_color}} !important;
+      {% endif %}
+    }
+  </style>
+{% endblock %}
+
+<div class="microsite">
+
+{% block notification_banner %}
+<!-- TODO: conditional notification banner on page model -->
+{% endblock %}
+
+{% block nav %}
+  {% if page.standard_header_show %}
+    {{ block.super }}
+  {% endif %}
+{% endblock %}
 
 {% block content %}
-{% if page.secondary_content_show %}
-<div style="background-color: #{% firstof page.secondary_content_background_color 'FFFFFF' %}">
-	{{ page.secondary_content | richtext }}
-</div>
-{% endif %}
+  <div id="primary_content" class="container-fluid microsite-content">
+    {% if page.custom_header_show %}
+      <div class="row">
+        <nav id="custom_header" class="navbar navbar--microsite pa20">
+          <div class="navbar__upper container">
+            {% if page.facebook_url or page.twitter_url %}
+              <div class="navbar__social">
+                {% if page.facebook_url %}
+                  <a href="{{ page.facebook_url }}" target="_blank" class="social-icon--facebook" onclick="trackOutboundLink('{{ page.facebook_url }}', true);">Facebook</a>
+                {% endif %}
+
+                {% if page.twitter_url %}
+                  <a href="{{ page.twitter_url }}" target="_blank" class="social-icon--twitter" onclick="trackOutboundLink('{{ page.twitter_url }}', true);">Twitter</a>
+                {% endif %}
+              </div>
+            {% endif %}
+
+            {% if page.custom_header_image %}
+              {% image page.custom_header_image max-600x600 as custom_header_image %}
+              <img src="{{ custom_header_image.url }}" alt="{{page.title}}" class="microsite-header-image">
+            {% endif %}
+
+            {% if page.button_url and page.button_text %}
+              <div class="navbar__buttons">
+                <a id="primary-cta"
+                  href="{{ page.button_url }}"
+                  target="{{ page.button_url_new_window|yesno:'_blank,_self'}}"
+                  class="btn btn-primary"
+                  onclick="trackOutboundLink('{{ page.button_url }}', {{ page.button_url_new_window|yesno:'true,false'}});">
```

Comment:
Suggest to just leave this enabled always. Field was not really meant for analytics so not sure this logic is desirable.

Reply:
What do you mean? This just changes the way we fire the event - if it's opened in new tab we have to do some custom logic which this code is for.

Reply:
Suggest only toggle new window based on field, but to trigger analytics events always (or after checking for external url). It's possible we might stay in same window and still go to external url.

Reply:
This does trigger analytics event always, the `true`/`false` just handles the event differently. If set to `true` but not to open in new window then event will still fire. If set to `false` and new window is open event will not properly trigger.